core/test/unit/web/parent-app_spec.js

```diff
@@ -0,0 +1,71 @@
+var should = require('should'),
+    sinon = require('sinon'),
+    proxyquire = require('proxyquire'),
+    sandbox = sinon.sandbox.create();
+
+describe('parent app', function () {
+    it('should successfully instantiate the app', function () {
+        should.exist(require('../../../server/web/parent-app')());
```

Comment:
Hmm you don't mock for this test. What was the reason? :) To ensure it does not crash?

Reply:
I don't have a strong opinion regarding this specific case either. I remember it was considered a good practice to instantiate a whole module/component in unit test back in Angular days. Will do some digging if there are similar cases elsewhere tomorrow morning.

Reply:
Decided to go without this specific unit test. Initialization would be tested anyways during other tests (integration, functional). Less code - better code :)